pkg/services/action.go

```diff
@@ -391,6 +391,10 @@ func NewCreateActionExtendLifecycleID(
 
 		intake.LifecycleExpiresAt = expirationDate
 
+		// TODO: set scope, next steps, etc. as well
```

Comment:
Will this TODO be resolved in this PR, or in another ticket?

Reply:
I was deciding on that myself, leaning towards another PR